mavros_msgs/CMakeLists.txt

```diff
@@ -1,14 +1,15 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(mavros_msgs)
 
-find_package(catkin REQUIRED COMPONENTS geometry_msgs message_generation std_msgs)
+find_package(catkin REQUIRED COMPONENTS geometry_msgs sensor_msgs message_generation std_msgs)
 
 include_directories(include)
 
 add_message_files(
   DIRECTORY msg
   FILES
   ActuatorControl.msg
+  ActuatorControlTarget.msg
```

Comment:
No! I don't like to break ROS API without strong reason. ActuatorControl here since 2015 and it does SET_ACTUATOR_CONTROL_TARGET for all that time.
And no reason to used different msg for return loop ACTUATOR_CONTROL_TARGET.